doc/classes/Rect2.xml

```diff
@@ -113,7 +113,7 @@
 			<argument index="0" name="by" type="float">
 			</argument>
 			<description>
-				Returns a copy of the [Rect2] grown a given amount of units towards all the sides.
+				Returns a copy of the [Rect2] grown [code]by[/code] the specified amount on all sides.
```

Comment:
That's not really understandable IMO to use code markup for the conjunction "by" here to refer to the parameter.

Reply:
It's an odd parameter name. Should we use the opportunity to rename the parameter name to `amount`?

Reply:
IMO "Returns of copy of the [Rect2] grown by the specified amount on all sides." is good enough, no need for custom markup. Otherwise it should be:  "Returns of copy of the [Rect2] grown by the specified [code]by[/code] amount on all sides.", or renaming "by" to something clearer like "offset".

Reply:
(Missed your comment as GitHub didn't refresh.)

Yes "amount" sounds fine too.

Reply:
Updated the variable name to `amount`.